src/main/java/com/xero/api/XeroRateLimitException.java

```diff
@@ -89,7 +89,7 @@ public Integer getMinuteLimitRemaining() {
     /** get retry after seconds
     * @return Long the number of seconds to wait before resuming API calls
     */
-    public long getRetryAfterSeconds() {
```

Comment:
if retryAfterSeconds is null (which it can be because the field type is "Long", this get method will try cast null to "long" (primitive long) and will result in NullPointerException